pkg/sql/parser/sql.y

```diff
@@ -1691,20 +1692,40 @@ import_format:
 import_stmt:
  IMPORT import_format '(' string_or_placeholder ')' opt_with_options
   {
+    /* SKIP DOC */
     $$.val = &tree.Import{Bundle: true, FileFormat: $2, Files: tree.Exprs{$4.expr()}, Options: $6.kvOptions()}
   }
+| IMPORT import_format DATA '(' string_or_placeholder ')' opt_with_options
```

Comment:
the `DATA` keyword is only used when the file is used only for its row-data. This one, and most of the ones duplicated below, are using the dump file for the schema and data, so they should not include the DATA keyword.

Reply:
I am sorry I am confused. Can you hand hold this for me?

The best would be for you to copy-paste the help text 10 lines above in your comment, and edit it yourself to show me what you mean.
Thank you

Reply:
I believe it would just be the original help text, with no changes -- ie. without the keyword `DATA`.

basically, the base case is `IMPORT PGDUMP 'my://dump/file.sql'` : this means open `file.sql` and import its contents.

If you didn't want all the contents, but rather only one table, you might use `IMPORT TABLE foo FROM PGDUMP 'my://dump/file.sql'`. This will open `file.sql` and import the table named "foo".

In both these cases, we read the definition of the table schemas from the file as well as the table data.

However, in some less common use cases, you might want or need to specify the schema yourself: this could be done to utilize cockroach-specific types, to control the conversion of Postgres or mysql types, or for file formats like CSV that do not contain the schema definition at all and only contain the row data.

For this case, we have the syntax `IMPORT TABLE foo (id int primary key, ...) CSV DATA ('my:///data/file.1.csv',...)`. This syntax allows specifying the schema separately from the files that contain the data. This is the only IMPORT syntax uses the keyword `DATA`, where it serves to indicate that the files are only used as data files, not full dump files that define the schema as well.